components/pricing_modal/content.tsx

```diff
@@ -140,6 +146,19 @@ function Content(props: ContentProps) {
         formatMessage({id: 'admin.billing.subscription.planDetails.features.mfa', defaultMessage: 'Multi-Factor Authentication (MFA)'}),
     ];
 
+    // Default professional price
+    const defaultProfessionalPrice = 10;
+    const [professionalPrice, setProfessionalPrice] = useState(defaultProfessionalPrice);
```

Comment:
May just want to set this to 0 in case the professional price changes in the future